tests/StaticTests.js

```diff
@@ -61,6 +68,8 @@ kettle.tests["static"].testDefs = [{
     },
     sequence: [{
         func: "{packageRequest}.send"
+    }, { // Send 2nd request back-to-back to test KETTLE-57
+        func: "{packageRequest2}.send"
```

Comment:
This test verifies that the request marker is cleared but not that it is set again at the middleware callback ("kettle.markActiveRequest(request)") -- is there any easy/convenient way to verify that the request is marked again upon return from async?